plugins/woocommerce-blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/block-settings/index.tsx

```diff
@@ -112,6 +112,7 @@ export const ProductGalleryThumbnailsBlockSettings = ( {
 					) }
 					max={ maxNumberOfThumbnails }
 					min={ minNumberOfThumbnails }
+					step={ 1 }
```

Comment:
Just curious what does the `step` prop do?

Reply:
According to the [docs](https://developer.wordpress.org/block-editor/reference-guides/components/range-control/#step-number-any):

> The minimum amount by which value changes. It is also a factor in validation as value must be a multiple of step (offset by min) to be valid. Accepts the special string value any that voids the validation constraint and overrides both withInputField and showTooltip props to false.

In this case, I put it more for safety reasons, considering that the minimum and maximum number of thumbnails could change, and the steps could be calculated around those values. I'm not entirely sure if this is necessary, but I'd like to guarantee that it will always be an integer.